styles.css

```diff
@@ -1 +1,72 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*instellingen voor desktops*/
+@media screen and (min-width: 576px) {
+    .container {
+        display: grid;
+        background-color: hsl(6, 72%, 89%);
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: 50px 190px 100px auto 50px;
+        grid-template-areas:
+            "blok-a blok-c blok-c"
+            "blok-b blok-c blok-c"
+            "blok-b blok-e blok-d"
+            "blok-b blok-e ."
+            "blok-f blok-f blok-f";
+        row-gap: 16px;
+        column-gap: 16px;
+    }
+}
+
+/*instellingen voor mobiele apparaten*/
+@media screen and (max-width: 576px) {
```

Comment:
In de uitwerking staat `(min-width: 576px)` ipv `(max-width: 576px)` Alles buiten de `min-width` is voor mobiel, nu is het andersom, alles buiten de `min-width` is voor desktop. Maakt eigenlijk niks uit.. bij deze :)